src/Features/LanguageServer/Protocol/Handler/Hover/HoverHandler.cs

```diff
@@ -47,17 +51,42 @@ public HoverHandler()
                 return null;
             }
 
-            var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
+            var supportsVSExtensions = context.ClientCapabilities.HasVisualStudioLspCapability() == true;
```

Comment:
`HasVisualStudioLspCapability` doesn't return a nullable bool does it?

Reply:
It did at some point, but looks like it doesn't any more!